server/src/moodRouter.mjs

```diff
@@ -37,6 +38,17 @@ moodRouter.get("/latest", async (request, response) => {
   console.log(request.user.sub);
 });
 
+// function called getQuote (take the mood and return a string)
+const url = "https://zenquotes.io/api/random";
+async function getQuote() {
+  const response = await fetch(url);
```

Comment:
Hmm, is there a reason why you set url as a variable and then call it in the fetch instead of using the actual url in the fetch fxn? 

I think if the url is not going to change, you can reduce the # of lines of code by hardcoding the url to the fetch fxn, but if there is a separate purpose then keep it as is.

```suggestion
  const response = await fetch("https://zenquotes.io/api/random");
```

Reply:
Great suggestion, thank you Serena!